modules/installation-custom-aws-vpc.adoc

```diff
@@ -96,6 +139,8 @@ C2S::
 ** `s3.<region>.c2s.ic.gov`
 endif::aws-secret[]
 
+When configuring the proxy in the `install-config.yaml` file, add these endpoints to the `noProxy` field. With this option, the proxy prevents the cluster from accessing the internet directly. However, network traffic remains private between your VPC and the required AWS services.
```

Comment:
@mjpytlak it looks like this part is missing in [1] and [2]

[1] http://file.rdu.redhat.com/mpytlak/osdocs3819/installing/installing_aws/installing-aws-user-infra.html
[2] http://file.rdu.redhat.com/mpytlak/osdocs3819/installing/installing_aws/installing-restricted-networks-aws.html

Reply:
@mjpytlak Just above one comment, the rest of contents LGTM.

Reply:
Nice catch @yunjiang29. Please take one more look.